orc8r/cloud/go/syncstore/store.go

```diff
@@ -65,3 +67,9 @@ type CacheWriter interface {
 	// Apply completes the batch cache update.
 	Apply() error
 }
+
+type DigestTrees map[string]*protos.DigestTree
+
+func (digestTrees DigestTrees) Networks() []string {
+	return funk.Keys(digestTrees).([]string)
```

Comment:
Nit: just construct this manually with a for loop, I've slowly turned away from over-using funk 🙂